### PR TITLE
cleaned up files a little

### PR DESCRIPTION
**Pull request by brooklynbowers**
your app should be in an 'app' folder, moved files around

### CODE REVIEW

File: .vscode/settings.json
Comment (leandrofahur) [inline .vscode/settings.json:1]: Try always to put those in a .gitignore file.

File: requirements.txt
Comment (leandrofahur) [inline requirements.txt:1]: I think you accidentaly added a character by the end

`pysqlite3-binary==0.5.2.post3ß`

**Review by leandrofahur — CHANGES_REQUESTED** [review]
Merge the current main with this branch, you will get the .gitignore file.

- Check the last line on file requirements.txt

**Review by daehako — COMMENTED** [review]
Approved.

**Pull request closed without merging**